lib/mllp/receiver.ex

```diff
@@ -79,6 +165,35 @@ defmodule MLLP.Receiver do
      ])}
   end
 
+  defp to_args(opts) do
```

Comment:
I like this, a clean approach to deal with all options. I like that we enforce now default dispatcher. I think, and as you also mentioned earlier, we should rename DefaultDispatcher to EchoDispatcher in a separate PR. In EchoDispatcher, we should remove `Logger.error` message. We should echo the message as info and returns positive ACK. It may require some changes in README as well.